TCCatServer/TCCatServer/Migrations/202007181048231_UpdateThreadUserIdColumnType.cs

```diff
@@ -0,0 +1,18 @@
+namespace TCCatServer.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class UpdateThreadUserIdColumnType : DbMigration
+    {
+        public override void Up()
+        {
+            AlterColumn("dbo.Threads", "UserId", c => c.String(nullable: false, maxLength: 128));
```

Comment:
Fixed one issue about the Thread model UserId field type, one mistake in last commit.